CDCgov/prime-reportstream
# 7572 Updates to HL7 to FHIR mappings for TQ1 segment

This PR updates the Service Request and Diagnostic Report resources  from HL7 v2 TQ1 segment using the latest HL7 v2 to FHIR mapping documents.

Copy of updated mapping document for TQ1
[HL7 to FHIR - TQ1 Segment.xlsx](https://github.com/CDCgov/prime-reportstream/files/10386794/HL7.to.FHIR.-.TQ1.Segment.xlsx)

Copy of updated mapping document for ORC
[HL7 to FHIR - ORC Segment.xlsx](https://github.com/CDCgov/prime-reportstream/files/10386876/HL7.to.FHIR.-.ORC.Segment.xlsx)

Test Steps:

1. Build this branch and run the unit and integration tests. Verify they all pass. Alternatively you can verify the Github Actions build passes. Note the HL7 to FHIR to HL7 integration test data was not changed, so the result of our HL7 to FHIR to HL7 conversion remains the same = no data changes.
2. Optional: Manually change the values for TQ1.7, TQ1.8, TQ1.9 and ORC.15 and verify that the values are being mapped correctly in the resulting FHIR file.

## Changes
- Updated HL7 v2 to FHIR mappings
- Updated FHIR to HL7 v2 mappings
- Updated FHIR files in integrations tests

## Checklist

### Testing

## Linked Issues
- Fixes #7572
